### DISCUSSION TITLE
where is the full dataset

### DISCUSSION THREAD

**Discussion by frankQDU** [PaddlePaddle/PaddleSpatial#15] [BaiduKDDCup2022] [answered]
Today is April 10. where is the full dataset

**Comment by zhoujingbo** [comment, marked as answer]
Dear frankQDU;

The full dataset will be released around 23:59  10 April on AOE time. Just wait for a while. 

Thanks a lot for your support.
Regards
Jingbo

**Reply by frankQDU** [reply]
Thank your answer.

**Reply by guyingyuexia** [reply]
I still haven't seen the dataset be released.

**Reply by frankQDU** [reply]
yes，you are right.
